### PR TITLE
Add missing ttl in pushhover

### PR DESCRIPTION
**Pull request by heliapb**
## Description
Add support ttl in pushover, following issue https://github.com/prometheus/alertmanager/issues/3454

### CODE REVIEW

File: notify/pushover/pushover.go [hunk 1]
```diff
@@ -147,6 +147,7 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 	parameters.Add("expire", fmt.Sprintf("%d", int64(time.Duration(n.conf.Expire).Seconds())))
 	parameters.Add("device", tmpl(n.conf.Device))
 	parameters.Add("sound", tmpl(n.conf.Sound))
+	parameters.Add("ttl", fmt.Sprintf("%d", int64(time.Duration(n.conf.TTL).Seconds())))
```
Comment (simonpasquier) [inline notify/pushover/pushover.go:150]: The parameter should be added only if the ttl is > 0.

Comment (heliapb) [reply notify/pushover/pushover.go:150]: Hi @simonpasquier could it be something like: 
```
	newttl := int64(time.Duration(n.conf.TTL).Seconds())
	if newttl > 0 {
		parameters.Add("ttl", fmt.Sprintf("%d", newttl))
	}
```

Comment (simonpasquier) [reply notify/pushover/pushover.go:150]: yes

Comment (heliapb) [reply notify/pushover/pushover.go]: ok thanks for the feedback, push the change, when you have the time pls review

File: docs/configuration.md [hunk 2]
```diff
@@ -945,6 +945,9 @@ token_file: <filepath>
 # acknowledges the notification.
 [ expire: <duration> | default = 1h ]
 
+# Optional time to live (ttl) to use for notification, see https://pushover.net/api#ttl
```
Comment (simonpasquier) [inline docs/configuration.md:948]:
```suggestion
# Optional time to live (TTL) to use for notification, see https://pushover.net/api#ttl
```

File: notify/pushover/pushover.go [hunk 3]
```diff
@@ -147,6 +147,14 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 	parameters.Add("expire", fmt.Sprintf("%d", int64(time.Duration(n.conf.Expire).Seconds())))
 	parameters.Add("device", tmpl(n.conf.Device))
 	parameters.Add("sound", tmpl(n.conf.Sound))
+
+	newttl := int64(time.Duration(n.conf.TTL).Seconds())
+
```
Comment (simonpasquier) [inline notify/pushover/pushover.go:152]:
```suggestion
```

**Review by simonpasquier — COMMENTED** [review]
2 small nits

**Review by simonpasquier — APPROVED** [review]

**Comment by heliapb** [comment]
hi @simonpasquier it failed the merge yesterday because it was missing a minor fix, could you check again pls?

**Comment by simonpasquier** [comment]
Thanks!

**Pull request merged**